tests/flytekit/unit/core/test_workflows.py

```diff
@@ -1,15 +1,24 @@
 import typing
 from collections import OrderedDict
 
+import pandas as pd
 import pytest
+from pandas._testing import assert_frame_equal
 
+from flytekit import kwtypes
 from flytekit.common.exceptions.user import FlyteValidationException, FlyteValueException
 from flytekit.common.translator import get_serializable
 from flytekit.core import context_manager
 from flytekit.core.condition import conditional
 from flytekit.core.context_manager import Image, ImageConfig
 from flytekit.core.task import task
 from flytekit.core.workflow import WorkflowFailurePolicy, WorkflowMetadata, WorkflowMetadataDefaults, workflow
+from flytekit.types.schema import FlyteSchema
+
+try:
+    from typing import Annotated
+except ImportError:
+    from typing_extensions import Annotated
```

Comment:
Do we need to do this? We're already pulling `typing_extensions` as part of a regular `flytekit` installation, right?

Reply:
hmm?  no we need this.  this is for compatibility between 3.7/8 and 3.9

Reply:
Why? We already have typing_extensions, so we could rely on that for all versions, including 3.9-10.

Reply:
From the user side, they may use `from typing import Annotated ` or `from typing_extensions import Annotated`. Therefore, we need to make sure both of them can work here.